docs_app/content/tutorial/3-creating-an-observable.md

```diff
@@ -0,0 +1,140 @@
+# 3. Creating An Observable
+
+## TLDR:
+
+- You should probably use a provided [creation function](API).
+- Create an observable with the `new Observable` [constructor](API).
+- The constructor wraps an initialization function that will be executed synchronously every time you call [`subscribe`](API).
+- The initialization function gives you a [`Subscriber`](API) instance to push [notifications](GL) to [consumers](GL).
+- The initialization function is expected to return [teardown](GL) logic, which is either a `() => void`, [`Subscription`](API), or `void` (if there is no teardown to be done).
+- When `complete` or `error` are called on the [`Subscriber`] provided by the initialization function, it will execute the provided [teardown](GL) as soon as possible.
+
+## Overview
+
+Besides consuming observables, another thing you'll likely need to do is create new observables. Creating an observable is relatively straight forward, as it is effectively wrapping a function to be executed later with some guarantees. But more importantly, RxJS provides a host of observable [creation functions](API) that create observables for you in tested ways.
```

Comment:
Consider elaborating on this point: "as it is effectively wrapping a function to be executed later with some guarantees" What does that mean, exactly? When later is it executed (on creation I presume)? And what types of guarantees?